src/components/Scrollable/Scrollable.tsx

```diff
@@ -31,6 +31,8 @@ export interface Props extends React.HTMLProps<HTMLDivElement> {
   shadow?: boolean;
   /** Slightly hints content upon mounting when scrollable */
   hint?: boolean;
+  // Fires when user reached the bottom of the scroll area
+  onScrollToBottom?: () => void;
```

Comment:
Nitpick: Our interfaces are typed like ⬇️ 
```suggestion
  onScrollToBottom?(): void;
```

Reply:
Right. Done.